### DISCUSSION TITLE
Adding Jules-Bertholet as collaborator for async-lock

### DISCUSSION THREAD

**Discussion by notgull** [smol-rs/smol#265] [General]
Hi,

I was checking contributions for `async-lock` for v2.8.0 and it looks like @Jules-Bertholet has written most of the code for this release. I was wondering if we wanted to offer them the opportunity for one of:

- Collaborator on `async-lock`
- Member of `smol-rs`

Any opinions on this? @smol-rs/admins

**Comment by zeenix** [comment]
@notgull I just have one question: Did they express a desire to be any of those? 

In any case, :+1: from me.

**Reply by notgull** [reply]
Not directly, I just wanted to make sure that we're all on the same case before I send the invite.
